### PR TITLE
Add first unit tests - panel

### PR DESCRIPTION
**Pull request by dglemos**
Test the following endpoints:
- `gene2phenotype/api/panels/`
- `gene2phenotype/api/panel/<str:name>/`
- `gene2phenotype/api/panel/<str:name>/summary/`

How to run tests:
`python manage.py test gene2phenotype_app.tests`

### CODE REVIEW

**Review by olaaustine — COMMENTED** [review]
Hi @dglemos, suggesting that we change this directory from test to test_views because this panel tests is for views section.

File: gene2phenotype_project/gene2phenotype_app/tests/panel.py [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+
+    def setUp(self):
+        self.url_panels = reverse('panel_details', kwargs={'name': 'DD'})
+
+    def test_get_panel_details(self):
+        """
+            Get the details for a visible panel.
+        """
+        response = self.client.get(self.url_panels)
+        self.assertEqual(response.status_code, 200)
```
Comment (olaaustine) [inline gene2phenotype_project/gene2phenotype_app/tests/panel.py:53]: Suggesting we should add another test here, that checks for a panelDetail maybe description

**Comment by dglemos** [comment]
> Hi @dglemos, suggesting that we change this directory from test to test_views because this panel tests is for views section and to follow Django's format we should rename the file from panel.py to test_panelviews.py if possible

Hi @olaaustine, the standard practice is to name the folder `tests`. Django won't automatically pick up the test files if we name the folder something other than `tests`.

File: gene2phenotype_project/gene2phenotype_app/tests/panel.py (hunk 1 above)
Comment (olaaustine) [reply gene2phenotype_project/gene2phenotype_app/tests/panel.py]: self.assertEqual(response.data.get('description'), 'Developmental disorders') - we can add this @dglemos

Comment (dglemos) [reply gene2phenotype_project/gene2phenotype_app/tests/panel.py]: Test added.

**Review by olaaustine — APPROVED** [review]
LGTM! Thank you @dglemos

**Pull request merged**